.github/pull_request_template.md

```diff
@@ -26,10 +19,12 @@ _Include any links here that might be helpful for people reviewing your PR. If t
 
 ## Output from acceptance tests
 
-_Please run the full suite of acceptance tests locally and include the output here._
+_Please run applicable acceptance tests locally and include the output here. If you are an external
+contributor, your contribution(s) will first be reviewed before running them against the project's
+CI pipeline._
 
 ```
-$ make testacc
+$ go test -v ./... -run TestAccTFEWorkspace # runs all workspace tests
```

Comment:
Nitpick: I don't think this command will actually run the acceptance tests unless the user has `TF_ACC=1` set!

```suggestion
$ TESTARGS="-run TestAccTFEWorkspace" make testacc 
```